sdk/communication/Azure.Communication.CallingServer/src/ServerCall.cs

```diff
@@ -241,7 +247,10 @@ public virtual async Task<Response<StartCallRecordingResult>> StartRecordingAsyn
         /// </summary>
         /// <param name="recordingStateCallbackUri">The uri to send state change callbacks.</param>
         /// <param name="cancellationToken">The cancellation token.</param>
-        public virtual Response<StartCallRecordingResult> StartRecording(Uri recordingStateCallbackUri, CancellationToken cancellationToken = default)
+        /// <param name="recordingContentType">Contenttype for recording.</param>
+        /// <param name="recordingChannelType">Channeltype for recording.</param>
+        /// <param name="recordingFormatType">Channeltype for recording.</param>
+        public virtual Response<StartCallRecordingResult> StartRecording(Uri recordingStateCallbackUri, RecordingContentType? recordingContentType = null, RecordingChannelType? recordingChannelType = null, RecordingFormatType? recordingFormatType = null, CancellationToken cancellationToken = default)
```

Comment:
RecordingContentType, RecordingChannelType and RecordingFormatType are currently in the Generated folder and will not be exposed in the SDK. To expose these models, create a class in the Models folder, similar to this class: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/communication/Azure.Communication.PhoneNumbers/src/Models/PhoneNumberCost.cs and then regenerate the code from swagger.